packages/terra-date-time-picker/src/DateTimeUtils.js

```diff
@@ -101,9 +105,10 @@ class DateTimeUtils {
    * @param {boolean} hasSeconds Whether or not seconds should be retrieved
    * @return {string} The time from the date and time string
    */
-  static getTime(time, hasSeconds) {
+  static getTime(time, hasSeconds, timeZone) {
     const timeFormat = hasSeconds ? 'HH:mm:ss' : 'HH:mm';
-    return DateUtil.formatISODate(time, timeFormat);
+    const momentDate = (timeZone && moment.tz.zone(timeZone) ? moment.tz(time, timeZone) : this.createSafeDate(time, timeZone));
```

Comment:
Are you able to just do `const momentDate = DateTimeUtils.createSafeDate(time, timeZone));` without the checks since createSafeDate already checks all that for you?

Reply:
udpated here 1e47747132f83358d17d2aaed1f98c6bd4a5cfaf